### DISCUSSION TITLE
storybook test interactions and resetting url args?

### DISCUSSION THREAD

**Discussion by ilovett** [storybookjs/storybook#27022] [Help]
### Summary

I have tests written that will pass on my first render, but after I leave and come back, the args are different values, using a render function with `useArgs`

How can we reset or configure ignoring URL args altogether?

I would like to always use the defined args in the story, and I don't give a crap about keeping the state if I navigate away and come back.

**Comment by jeremydrichardson** [comment]
I would like to know this as well. 

This breaks interaction tests if the the interaction is assuming a default starting point. However, if it's not then the interaction test fails because it gets something different than it was expecting.

Maybe there could be a way from the play function to reset all args?

**Comment by squidjam** [comment]
Having means to reset play functions or to explicitly have means to avoid them through test-runner and some tags could be great.
I don't think duplicating stories or having to write actions that reset the interactions is a good idea...
